src/common/gfx.cpp

```diff
@@ -577,47 +578,47 @@ void gfx_drawpreview(
     }
 }
 
-bool gfx_loadteamcoloredimage(gfxSprite * gSprites, const std::string& filename, bool fVertical, bool fWrap)
+bool gfx_loadteamcoloredimage(gfxSprite* sprites, const std::string& path, bool fVertical, bool fWrap)
 {
-    return gfx_loadteamcoloredimage(gSprites, filename, 255, 0, 255, 255, fVertical, fWrap);
+    return gfx_loadteamcoloredimage(sprites, path, colors::MAGENTA, 255, fVertical, fWrap);
```

Comment:
I'm nitpicking, but it being magenta is probably not the important thing about 0xff00ff in this case. What is important is that it is the background color.

Reply:
You are correct, and this is why the parameter name in `gfxSprite::init` is `key` (from color key) instead of just RGB. However, some images use a different color key, for example black; calling 0xff00ff then as 'transparent' would be misleading, so I went with the color names.